euclidlike_imsim/noise.py

```diff
@@ -97,8 +111,16 @@ def get_noise(cfg_noise, cfg_image, base, logger):
     # Make integer ADU now.
     noise_img.quantize()
 
+    # Add quantization noise (this avoid issues related to rounding e.g.
+    # background estimation)
+    if cfg_noise["quantization_noise"]:
+        quantization_noise = noise_img.copy()
+        quantization_noise.fill(0)
+        quantization_noise.addNoise(galsim.DeviateNoise(galsim.UniformDeviate(rng)))
```

Comment:
On the github interface something funny is happening with indentation?  Is this a tabs vs. spaces thing or just a fluke?